src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,35 @@
+.card__stars {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0 0 24px;
```

Comment:
.card__stars should be inside the .card selector. Leave only stars here and move everything connected with `card` to the card block

Reply:
done